frontend/src/views/Combat.vue

```diff
@@ -137,6 +140,7 @@ export default {
       intervalMinutes: null,
       timeSeconds: null,
       timeMinutes: null,
+      fightXpGain: 32,
```

Comment:
XP is not guaranteed to be 32; this needs to be correctly calculated - if I remember right it's from 14-17, but that could be clarified.

Reply:
who should we ask... i saw that number in cryptoblades.sol tho it looks like its some sort  of  special case
```
    function migrateTo_1ee400a() public {
        require(hasRole(DEFAULT_ADMIN_ROLE, msg.sender), "Not admin");

        fightXpGain = 32;
    }
```